ext/handlers_http.h

```diff
@@ -75,6 +75,11 @@ static inline zend_string *ddtrace_format_tracestate(zend_string *tracestate, ze
         smart_str_append(&str, tracestate);
     }
 
+    if (str.s && ZSTR_VAL(str.s)[ZSTR_LEN(str.s) - 1] == ',') {
+        ZSTR_VAL(str.s)[ZSTR_LEN(str.s) - 1] = '\0';
+        ZSTR_LEN(str.s)--;
+    }
+
```

Comment:
Note to self: This is really caused by bad handling of the trailing comma in distributed tracing _parsing_ and has been fixed ... on the span links branch.